feat: add plugins-benchmark-pr.yml (#90)
fastify/workflows: README.md

| `license-check-allowed-additional` | false      | string  |           | Provide a semicolon separated list of SPDX-license identifiers that you want to additionally allow. |
| `lint`                             | false      | boolean | `false`   | Set to `true` to run the `lint` script in a repository's `package.json`.           |

> ## Benchmark PR-Workflow

Reviewer: ```suggestion
## Benchmark PR workflow
```

> The benchmark workflow expects pull_request or pull_request_target events. Most common use is to run the benchmarks when a `benchmark` label is added to the PR.

Author: ```suggestion
The benchmark workflow expects `pull_request` or `pull_request_target` events. A common use for this workflow is to run benchmarks when a `benchmark` label is added to the PR.
```
